text/007-define-rules.md

```
@@ -0,0 +1,1041 @@
+ Start Date: 2025-11-16
+
+# Summary
+[summary]: #summary
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "summary"

(MD053, link-image-reference-definitions)